pkg/server/httpAPI.go

```diff
@@ -68,8 +69,8 @@ func (h *HTTPHandler) Stop() error {
 // ServeHTTP serves the http re
 func (h *HTTPHandler) serveMetrics(rw http.ResponseWriter, req *http.Request) {
```

Comment:
Please rename endpoint to `Healthz` instead of `Metrics`.  Please mention it in `CHANGELOG.md` as well